src/extension/content-script/batteries/YouTubeVideo.ts

```diff
@@ -23,6 +23,7 @@ const battery = async (): Promise<Battery | void> => {
   }
   let match;
   let lnurl;
+  const contentUri = document.location.toString();
```

Comment:
I think we should extract the canonical url from the html document. The location includes query parameters and other things which can produce a different content URI.

Note: if we decide not to include contentUri and have a different schema per content type, then we could just include the video ID.

Reply:
resolved